### DISCUSSION TITLE
Para front end: Alura ou Danki Code?

### DISCUSSION THREAD

**Discussion by lucaspacola** [frontendbr/forum#1793] [Discussão]
Boa noite pessoal,

Sou novato no universo da programação e estou focado em aprender frond end, recentemente conclui o curso do Curso em vídeo com o Guanabara (excelente curso por sinal). Porém, gostaria de aprofundar, agora que já aprendi o básico, e encontrei dois cursos que me parecem bem completos em duas instituições diferentes. Alura e Danki Code, alguém tem algum feedback de uma delas ou até de ambas para me ajudar na escolha?

Agradeço

**Comment by victorferreira** [comment]
Não conheço o Danki Code, mas o curso de JS avançado da Alura, com @flaviohenriquealmeida, foi um dos melhores materiais sobre JS que eu já consumi sobre o assunto, levando em conta material brasileiro e gringo. O curso de React deles não era tão legal, mas o de JS avançado valeu a assinatura de um ano que paguei.

**Comment by vagnerlandio** [comment]
Concordo com o @victorferreira também fiz o curso de front-end lá na Alura, desde de o HTML até o JS avançado e o de React, não aprendi muito com o curso de React, mas lembro que quando terminou minha assinatura eles estavam renovando o curso de React.

Sobre o de JS, consegui desenvolver uma PWA escrita somente com JS puro e organizada em MVC e conectado com banco de dados local chamado PouchDB.

Valeu muito apena a Alura.

**Comment by wilsonneto-swe** [comment]
Não fiz tantos cursos, alura já ouvi falar bem sim, mas eu sempre busquei cursos focados, os dois que fiz que me ajudaram demais na carreira foi da Rocketset e do Dev Pleno, o dev pleno me deu uma base já evoluindo pro lado do react, gostei bastante, mas depende do seu objetivo.

Rocketseat as inscrições estão fechadas,
DevPleno para saber mais: http://bit.ly/reactjs-dev-pleno

Espero que ajude, abraço!

**Comment by yuritoledo** [comment]
Se você não ler bem o inglês, algum curso da Udemy, que sempre está em promoção por 20 reais, como esse: https://www.udemy.com/course/curso-web/

Ou se manjar de inglês, fala o freecodecamp, é grátis e é o mais completo que já vi: https://www.freecodecamp.org/

**Comment by rtcorradi** [comment]
Tenho curso da Danki Code e não gostei muito da didática comparado a outros cursos, as aulas não são bem produzidas, parece que ele abre o editor e vai falando, gosto dos cursos da Cod3r do Leonardo Leitão, que estão na Udemy com um preço mais acessível, ouço falar bem da Alura tbm.
